sentry/src/test/java/io/sentry/HubTest.kt

```diff
@@ -424,43 +366,73 @@ class HubTest {
 
     @Test
     fun `when captureException is called with a valid argument and hint, captureException on the client should be called`() {
-        val options = SentryOptions()
-        options.cacheDirPath = file.absolutePath
-        options.dsn = "https://key@sentry.io/proj"
-        options.setSerializer(mock())
-        val sut = Hub(options)
-        val mockClient = mock<ISentryClient>()
-        sut.bindClient(mockClient)
+        val (sut, mockClient) = getEnabledHub()
 
         sut.captureException(Throwable(), Object())
         verify(mockClient).captureException(any(), any(), any())
     }
 
     @Test
     fun `when captureException is called with a valid argument but no hint, captureException on the client should be called`() {
-        val options = SentryOptions()
-        options.cacheDirPath = file.absolutePath
-        options.dsn = "https://key@sentry.io/proj"
-        options.setSerializer(mock())
-        val sut = Hub(options)
-        val mockClient = mock<ISentryClient>()
-        sut.bindClient(mockClient)
+        val (sut, mockClient) = getEnabledHub()
 
         sut.captureException(Throwable())
         verify(mockClient).captureException(any(), any(), isNull())
     }
     //endregion
 
+    //region captureUserFeedback tests
+
+    @Test
+    fun `when captureUserFeedback is called it is forwarded to the client`() {
+        val (sut, mockClient) = getEnabledHub()
+        sut.captureUserFeedback(userFeedback)
+
+        val captor = ArgumentCaptor.forClass(UserFeedback::class.java)
+        verify(mockClient).captureUserFeedback(captor.capture())
```

Comment:
M: I'm not a big fan of `ArgumentCaptor` tbh, what's about doing this with argument matches?

https://github.com/nhaarman/mockito-kotlin/wiki/Mocking-and-verifying#argument-matchers

the testing classes today uses either `check` or `argThat` I guess